src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs

```diff
@@ -141,7 +141,7 @@ static void DrawBackground(CGContext context, Paint? brush, CGSize imageSize)
 		switch (brush)
 		{
 			case SolidPaint solidColorBrush:
-				context.SetFillColor(solidColorBrush.Color.ToCGColor());
+				context.SetFillColor(solidColorBrush.Color?.ToCGColor() ?? DrawingViewDefaults.BackgroundColor.AsCGColor());
```

Comment:
use DrawingView default background color if Background is not defined

Reply:
Is it ok on the other platofrms?

Reply:
that was the reason I added do-not-merge tag.
on android Extension method excepts nullable. so it works.
on Windows I added additional check